MaxText/pyconfig.py

```diff
@@ -196,6 +196,8 @@ def user_init(raw_keys):
     raw_keys['num_kv_heads'] = 2**num_head_scale * raw_keys['base_num_kv_heads']
     raw_keys['mlp_dim'] = 2**mlp_dim_scale * raw_keys['base_mlp_dim']
     raw_keys['num_decoder_layers'] = 2**layer_scale * raw_keys['base_num_decoder_layers']
+    raw_keys['num_experts'] = raw_keys['num_experts']
```

Comment:
probably not necessary? Generally I don't think this should do anything or be needed?

Reply:
Yes, that's right! Thanks for catching!